src/main/resources/alma/fix/subjects.fix

```diff
@@ -296,8 +296,10 @@ do list(path:"084??", "var":"$i")
             if any_match("$j","\\d{3}(\\.\\d{1,3})?|[BKS]")
               if any_equal("$i.q","DE-600")
                 add_field("subject[].$last.source.label","DDC-Sachgruppen der ZDB")
+                add_field("subject[].$last.source.id","https://zeitschriftendatenbank.de/fileadmin/user_upload/ZDB/pdf/zdbformat/5080.pdf")
```

Comment:
Interesting choice of `id` for this source. While I like the pragmatism, this sets a new precedent linking to the description of a controlled vocabulary in a documentation PDF. As I have no better idea, I guess we can leave it for now.

Reply:
On the other hand, we should probably better leave out the `id` field and add the documentation link in another field, e.g. `seeAlso`.

Reply:
I already had a bad feeling a while ago about id: https://github.com/hbz/lobid-resources/issues/1848#issuecomment-1707846009

Reply:
But we decided off board to go on for a lack of a better id.

Reply:
Ok, then let's go. +1